src/main/java/core/basesyntax/Buldoser.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Buldoser extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Buldoser started working");
```

Comment:
The message in the 'doWork' method should be more informative. For example, it could be 'Bulldozer started its work.' to clearly indicate that the Bulldozer is the machine currently working.